src/utils/index.ts

```diff
@@ -79,15 +77,23 @@ export function getMatchedComponents(
   )
 }
 
-export function normalizePath(path = ''): string {
+export function normalizePath(path = '', ctx?: Context): string {
   // Remove query string
   let result = path.split('?')[0]
 
+  // Remove base path
+  if (ctx && ctx.base) {
```

Comment:
I would split this out of `normalizePath` into a utility like `withoutBase(path, base)`. Also would be nice at some point we migrate to [ufo](https://github.com/nuxt-contrib/ufo) since it is standard library we started using in several places

Reply:
I think I'll let this migration with you, as you is the author of ufo and I've not used it yet.

Reply:
Yeah, good idea! I agree about splitting into `withoutBase`. But should I do it tomorrow or wait for ufo migration?